### PR TITLE
technology auto guidebook

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

adds a guidebook page with all technologies.
we have the techmology

Also updates the research console tooltips + makes them colorblind accessible. Whoops.

## Media

image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/Science/Science.xml [hunk 1]
```diff
@@ -12,6 +12,8 @@ The most important thing inside your department is the R&D server, which stores
 
 Each technology costs [color=#a4885c]Research Points[/color] and unlocks recipes at lathes. Some technologies will also have prerequesites you have to unlock before you can research them.
 
+The technologies can be viewed [textlink="here" link="Technologies"].
+
```
Comment (metalgearsloth) [inline Resources/ServerInfo/Guidebook/Science/Science.xml:16]: For accessibility reasons you should avoid linking something as "here" and it should be somewhat self-descriptive, e.g. "The technologies can be viewed on the <Technologies page>" or just "here <link>".

File: Content.Client/Guidebook/Controls/GuideTechDisciplineEmbed.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            DisciplineContainer.AddChild(embed);
+        }
+    }
+
+    public bool TryParseTag(Dictionary<string, string> args, [NotNullWhen(true)] out Control? control)
+    {
+        control = null;
+        if (!args.TryGetValue("Discipline", out var group))
+        {
+            Logger.Error("Reagent group embed tag is missing group argument");
```
Comment (moonheart08) [inline Content.Client/Guidebook/Controls/GuideTechDisciplineEmbed.xaml.cs:45]: copy paste 🏏

**Review by moonheart08 — APPROVED** [review]
TECHNOLOGY

File: Resources/ServerInfo/Guidebook/Science/Science.xml [hunk 3]
```diff
@@ -10,9 +10,9 @@ Science, often called Research and Development, is a job made up of both generat
 </Box>
 The most important thing inside your department is the R&D server, which stores unlocked technologies, and the R&D computer, which allows you to unlock technologies.
 
-Each technology costs [color=#a4885c]Research Points[/color] and unlocks recipes at lathes. Some technologies will also have prerequesites you have to unlock before you can research them.
+Each technology costs [color=#a4885c]Research Points[/color] and unlocks recipes at lathes. Some technologies will also have prerequisites you have to unlock before you can research them.
 
-The technologies can be viewed [textlink="here" link="Technologies"].
+Information about the differenet technologies can be viewed [textlink="on the technology guidebook page" link="Technologies"].
```
Comment (moonheart08) [inline Resources/ServerInfo/Guidebook/Science/Science.xml:15]: Differenet

**Pull request merged**